doc/other_modules/forge2d.md

```diff
@@ -65,57 +65,55 @@ not according to the coordinate system of Forge2D (where the Y-axis is flipped).
 
 ## Contact callbacks
 
-If you are using `Forge2DGame` you can take advantage of its way of handling contacts between two
-`BodyComponent`s.
+`Forge2DGame` provides a simple out of the box solution to propagate contact events.
 
-When creating the body definition for your `BodyComponent` make sure that you set the userdata to
-the current object, otherwise it will not be possible to detect collisions.
-Like this:
-```dart
-final bodyDef = BodyDef()
-  // To be able to know which component that is involved in a collision
-  ..userData = this;
-```
+Contacts events occur whenever two `Fixture`s meet each other. These events allows listening when
+these `Fixture`s begin to overlap (`beginContact`) and cease overlapping (`endContact`).
```

Comment:
```suggestion
Contact events occur whenever two `Fixture`s meet each other. These events allows listening when
these `Fixture`s begin to come in contact (`beginContact`) and cease being in contact
(`endContact`).
```

Reply:
I prefer `overlap` since "sensors" don't actually contact. I believe the official Box2D documentation also uses "overlap".

Refer to: https://box2d.org/documentation/md__d_1__git_hub_box2d_docs_dynamics.html#autotoc_md107

Reply:
I think it is grammatically correct to say that they do come in contact with each other still, even if it is an overlap when they do. Idk, but I think "coming in contact" is clearer to the user than overlap, even though that is what actually happens (even for solid bodies).

Reply:
Indeed, it depends on how technical and specific we want to be since we could say that there is always an overlap (AAAB).

Reply:
I don't think that is needed on this level of documentation. There could be a section about the actual technical solution later if needed, but this should be user documentation so it should be as easy to understand as possible, while still being technically correct of course.